tests/integrations/controllers/WelcomeController.py

```diff
@@ -14,27 +14,24 @@ def test(self):
     def view(self, view: View):
         return view.render("welcome")
 
-    def create(self, view: View):
-        return view.render("welcome"), 201
+    def create(self):
+        return "user created", 201
 
     def not_found(self):
         return "not found", 404
 
-    def unauthorized(self, view: View):
+    def unauthorized(self):
         return "unauthorized", 401
 
-    def forbidden(self, view: View):
-        return view.render("forbidden"), 403
+    def forbidden(self):
+        return "forbidden", 403
```

Comment:
love this new response. @josephmancuso

Reply:
yes very clean :)

Reply:
This s actually possible in Masonite 3 too lol